packages/eslint-plugin/src/rules/lines-between-class-members.ts

```diff
@@ -9,16 +9,20 @@ const baseRule = getESLintCoreRule('lines-between-class-members');
 type Options = util.InferOptionsTypeFromRule<typeof baseRule>;
 type MessageIds = util.InferMessageIdsTypeFromRule<typeof baseRule>;
 
-const schema = util.deepMerge(
-  { ...baseRule.meta.schema },
-  {
-    1: {
-      exceptAfterOverload: {
-        type: 'boolean',
-        default: true,
+const schema = Object.values(
```

Comment:
Before, we had:

```ts
{ "0": { /* schema A */ }, "1": { /* schema B */ } }
```

AJV is actually not able to interpret this correctly, and no validation actually occurs.

This change fixes it to:

```ts
[{ /* schema A */ },  { /* schema B */ }]
```

Using Object.values here should be safe. The baseRule.meta.schema is an array itself, so will be spread in order (guaranteed by ECMA spec). deepMerge will merge the keys in the order of the first object. Object.values will take the values in order, so we'll get back the array in the right order.

Reply:
Nifty trick to use `Object.values` to convert the string index style object to an array!